woocommerce/woocommerce
# Remove pull_request trigger for metrics workflow

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Remove the `pull_request` trigger for the `Metrics Tracking` workflow to further optimize the runners usage. The workflow remains active for push to trunk, event for which the performance results are actually published.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Check CI. `Metrics Tracking` workflow should not run.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
